src/reducers/versions.spec.tsx

```diff
@@ -1774,4 +1868,45 @@ describe(__filename, () => {
       expect(state.versionInfo[beginFetchVersionId]).toEqual(undefined);
     });
   });
+
+  describe('getCompareInfoKey', () => {
+    it('computes a key given an addonId, baseVersionId and headVersionId', () => {
+      const addonId = 123;
+      const baseVersionId = 1;
+      const headVersionId = 2;
+
+      expect(
+        getCompareInfoKey({ addonId, baseVersionId, headVersionId }),
+      ).toEqual(`${addonId}/${baseVersionId}/${headVersionId}/`);
+    });
+
+    it('computes a key given an addonId, baseVersionId, headVersionId and path', () => {
+      const addonId = 123;
+      const baseVersionId = 1;
+      const headVersionId = 2;
+      const path = 'path';
+
+      expect(
+        getCompareInfoKey({ addonId, baseVersionId, headVersionId, path }),
+      ).toEqual(`${addonId}/${baseVersionId}/${headVersionId}/${path}`);
+    });
+  });
+
+  describe('isCompareInfoLoading', () => {
+    it('returns false by default', () => {
```

Comment:
This should have an additional test for `true`. Even though other tests will fail, it will be helpful to have a specific failure to help track down the root cause.